.travis.yml

```diff
@@ -137,7 +142,6 @@ jobs:
         - echo "Releasing $MEDEA_CRATE_NAME to crates.io..."
       deploy:
         provider: script
-        skip_cleanup: true
```

Comment:
@evdokimovs @alexlapa why this was removed?

Reply:
@tyranron ,

https://docs.travis-ci.com/user/deployment-v2#cleaning-up-the-git-working-directory

Reply:
Please, describe motivation for such changes in PR'sdescription. Otherwise, it's too hard to understand.